src/ProjectTemplates/test/WorkerTemplateTest.cs

```diff
@@ -21,13 +21,18 @@ public WorkerTemplateTest(ProjectFactoryFixture projectFactory, ITestOutputHelpe
         public ProjectFactoryFixture ProjectFactory { get; }
         public ITestOutputHelper Output { get; }
 
-        [ConditionalFact]
+        [ConditionalTheory]
         [OSSkipCondition(OperatingSystems.Linux, SkipReason = "https://github.com/dotnet/sdk/issues/12831")]
-        public async Task WorkerTemplateAsync()
+        [TheoryData("C#")]
```

Comment:
InlineData doesn't work here?

Reply:
Yes, it works. I just did the wrong thing. Will fix it tomorrow&hellip;